.travis.yml

```diff
@@ -1,9 +1,14 @@
 language: python
-python: 2.7
-env:
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
+matrix:
+  include:
+    - python: "2.7"
+      env: TOX_ENV=py27
+    - python: "3.3"
+      env: TOX_ENV=py33
+    - python: "3.4"
+      env: TOX_ENV=py34
+    - python: "3.5"
+      env: TOX_ENV=py35
```

Comment:
This looks more complicated than necessary, but it's the one of the ways to guarantee Travis will find the correct interpreter. You'll find a lot of issues about this if you google "Travis InterpreterNotFound"; see [this issue](https://github.com/travis-ci/travis-ci/issues/4794) among many others.